src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -169,15 +168,17 @@ public ImageSource LargeIcon
             }
         }
 
+        public Point Position;
+
```

Comment:
This should have been made a property instead, just like `ClickedCommand` below it, and then made `internal`.